### ISSUE TITLE
Stacked widget dont work

### ISSUE THREAD

**Issue by Oberonhive** [jupyter-widgets/ipywidgets#2789]
On latest version this code produces error in chrome console.

```
import ipywidgets
ipywidgets.Stacked()
```
Error: `Cannot find model module @jupyter-widgets/controls@1.5.0, StackedModel`

Tried regenerate model schema (generate-spec.py) with no result.

**Comment by SylvainCorlay** [comment]
Stacked is not in the latest stable version, but only in the master branch to be released as 8.0.

Hence this is expected behavior.

**Comment by SylvainCorlay** [comment]
Also, you appear to be installing the python side from source right?

**Comment by Oberonhive** [comment]
Stacked widget looks extremely usefull in my case, I hope it will be fixed soon. Anyway thanks for great job, ipywidges saves a ton of time )

> Also, you appear to be installing the python side from source right?
Yes. It's installed by this script.
```
git clone https://github.com/jupyter-widgets/ipywidgets.git
cd ipywidgets
yarn
yarn run clean
yarn run build 
pip install -e .
```

**Comment by YoraiLevi** [comment]
> Stacked widget looks extremely usefull in my case, I hope it will be fixed soon. Anyway thanks for great job, ipywidges saves a ton of time )
> 
> > Also, you appear to be installing the python side from source right?
> > Yes. It's installed by this script.
> 
> ```
> git clone https://github.com/jupyter-widgets/ipywidgets.git
> cd ipywidgets
> yarn
> yarn run clean
> yarn run build 
> pip install -e .
> ```

@Oberonhive  Did you get the Stacked to work? I am trying to run the [example in the docs ](https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20List.html#Stacked)under google colab with the setup script
```bash
#!wget https://github.com/jupyter-widgets/ipywidgets/archive/8.0.0a0.tar.gz
#!tar -C ipywidgets/ -xvf 8.0.0a0.tar.gz
!git clone https://github.com/jupyter-widgets/ipywidgets.git
%cd ipywidgets
!apt install node
!apt install yarn
!yarn
!yarn run clean
!yarn run build 
!pip install -e .
%cd /content
```
> restart the runtime as the colab button suggests
```bash
!pip show ipywidgets
```

> Name: ipywidgets
Version: 8.0.0a0
Summary: Jupyter interactive widgets
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /content/ipywidgets
Requires: ipykernel, ipython, traitlets, nbformat, widgetsnbextension
Required-by: jupyter, cufflinks

```python
import ipywidgets as widgets

button = widgets.Button(description='Click here')
slider = widgets.IntSlider()
stacked = widgets.Stacked([button, slider])
stacked
```
stacked isn't being shown(white):
![image](https://user-images.githubusercontent.com/50873841/91353352-d2ec6b80-e7f3-11ea-86dd-7598e11df502.png)

**Comment by amaurydar** [comment]
The Stacked widget still doesn't seem to work in 8.0.0a6, is this expected ?

In particular it doesn't work in the binder mentioned in the doc :
https://mybinder.org/v2/gh/jupyter-widgets/ipywidgets/8.0.0a6?urlpath=lab/tree/docs/source/examples/Widget%20List.ipynb

**Comment by shadiakiki1986** [comment]
@YoraiLevi ipywidgets.Stacked not working in colab is another issue: https://github.com/googlecolab/colabtools/issues/498#issuecomment-956321234

**Comment by blois** [comment]
Colab's in-progress support for 3rd party widgets builds off of the ipywidgets 8.0 alpha so it can be used- an example is:
https://colab.research.google.com/gist/blois/2eed369f917b6e71a717684a65a06a84/stacked_widget.ipynb

I'm seeing an error when the `selected_index` is not specified for `widgets.Stacked`. I believe it comes from `null` being allowed here: https://github.com/jupyter-widgets/ipywidgets/blob/51cb73a0b31ed3f087de52d5122e565aa47b45f6/ipywidgets/widgets/widget_selectioncontainer.py#L19

But then that doesn't appear to be handled here:
https://github.com/jupyter-widgets/ipywidgets/blob/51cb73a0b31ed3f087de52d5122e565aa47b45f6/packages/controls/src/widget_selectioncontainer.ts#L442-L445

(But this error may be Colab-specific)
